benchmarks/aes/yamls/docker-compose/dc-aes-go.yaml

```diff
@@ -25,6 +25,19 @@ services:
   aes-go:
     image: vhiveease/aes-go:latest
     container_name: aes-go
+    environment:
+      - COMPOSE_PROJECT_NAME=aes-go
     ports:
-      - published: 50051
-        target: 50051
+      - target: 50051
```

Comment:
It will be fine to remove published for aes servers right because we want only relay to be exposed (and zipkin)? I will remove published from elsewhere also then.